docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -173,6 +173,17 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 - `BLACKLISTED_URIS`: **\<empty\>**: If non-empty, list of POSIX regex patterns matching
    OpenID URI's to block.
 
+## OAuth2 (`oauth2`)
+
+- `OAUTH2_REGISTER_EMAIL_CONFIRM`: **REGISTER\_EMAIL\_CONFIRM**: Set this to enable or disable
+   mail confirmation of OAuth2 auto-registration.
```

Comment:
can we spell this out as email instead of mail?

Reply:
Fixed in 6a3a0e7.